mutation_calling/fusion/fusion-inspector/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# ProTECT Dockerfile for STAR-Fusion
+#
+# Build with: sudo docker build --force-rm --no-cache -t jpfeil/fusion-inspector:1.0.1 - < Dockerfile
+# Run with sudo docker run -v <IO_folder>:/data jpfeil/fusion-inspector:1.0.1 <parameters>
+
+# Use pimmuno
+FROM aarjunrao/pimmuno
+
+MAINTAINER Jacob Pfeil, jpfeil@ucsc.edu
+
+RUN apt-get update --fix-missing && \
+    apt-get install -y dpkg zlib1g-dev gzip perl libdb-dev build-essential make
+
+RUN echo "deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | tee /etc/apt/sources.list.d/webupd8team-java.list && \
+    echo "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main" | tee -a /etc/apt/sources.list.d/webupd8team-java.list && \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys EEA14886 && \
+    apt-get update && \
+    echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections && \
+    apt-get install -y oracle-java8-installer
+
+RUN cpan App::cpanminus && cpanm Set::IntervalTree && cpanm DB_File && cpanm URI
+
+WORKDIR /opt
+
+# Download Wrappers 
+RUN git clone https://github.com/jpfeil/toil_docker_wrappers.git /opt/docker_wrappers
```

Comment:
wrappers go to /home. Also add the wrappers to my toil_docker_wrappers repo to maintain homogeneity.